main.go

```diff
@@ -223,10 +224,19 @@ func main() {
 		log.Fatalf("Unable to initialize the rasterizer cache: %s", err)
 	}
 
-	// Set up an S3-backed filecache to underly the rasterCache
-	fCache, err := filecache.NewS3Cache(
-		config.CacheSize, config.BaseDir, config.AwsRegion, ServerWriteTimeout,
-	)
+	// Set up a filecache to underly the rasterCache
+	var fCache *filecache.FileCache
+	if config.DownloadManager == "s3" {
```

Comment:
Are you planning to run separate instances of this service for each kind of backend? Again, hardcoding download manager names here is a smell. I'd have a function that returns the right thing or at a minimum a const with iota to reference them.

Reply:
Yep, we want to have separate dedicated instances, so I think a runtime flag is in order here. Const & iota sounds good.